src/includes/configure-scope/dart.mdx

```diff
@@ -0,0 +1,9 @@
+```dart
+import 'package:sentry/sentry.dart';
+
+void main() {
+  Sentry.configureScope((scope) => scope
+    ..setTag('my-tag', 'my value')
+    ..user = User(id: '42', email: 'john.doe@example.com'));
+}
```

Comment:
I suggest we delete `main` from all examples (exccept the `init` example)

it's confusing as it gives the impression copy pasting this would work but it wouldn't. Because it doesn't have `init` right after `main`.

```suggestion
Sentry.configureScope((scope) => scope
  ..setTag('my-tag', 'my value')
  ..user = User(id: '42', email: 'john.doe@example.com'));
```

Reply:
removed them